src/Infrastructure/Repositories/EmployeeRepository.cs

```diff
@@ -173,7 +179,8 @@ public async Task AddOrUpdateCvInformation(List<Cv> cvs)
     {
         foreach (Cv cv in cvs)
         {
-            var entity = await GetEmployeeEntityWithCv(cv.Email);
+            _logger.LogInformation("Starting cv processing of " + cv.Email);

```

Comment:
Har vi en end?

Reply:
Ikke for hver enkel CV.  Det er en end log som kalles når alle er prosessert. Ikke inni loopen. Den hopper jo bare rett til neste , så ser ikke helt poenget.